src/xrpld/app/tx/detail/PayChan.cpp

```diff
@@ -516,18 +537,12 @@ PayChanClaim::doApply()
             (txAccount == src && (sled->getFlags() & lsfDisallowXRP)))
             return tecNO_TARGET;
 
-        // Check whether the destination account requires deposit authorization.
-        if (depositAuth && (sled->getFlags() & lsfDepositAuth))
+        if (depositAuth)
         {
-            // A destination account that requires authorization has two
-            // ways to get a Payment Channel Claim into the account:
-            //  1. If Account == Destination, or
-            //  2. If Account is deposit preauthorized by destination.
-            if (txAccount != dst)
-            {
-                if (!view().exists(keylet::depositPreauth(dst, txAccount)))
-                    return tecNO_PERMISSION;
-            }
+            if (auto err = credentials::verify(
```

Comment:
Same comment as in escrow. Difficult to compare pre/post amendment code.

Reply:
smae here